### DISCUSSION TITLE
Thinking Cap #1

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/22S-ASIAAM-191A#1] [Assignments]
#### Note: responses submitted after Sunday 2pm (4/3) will not be eligible for extra credit.

> ## What do you think are problems with the way we use data and GIS?

Post your thinking cap response and reply to others in this thread here. 

##### Refer to this [documentation](https://albertkun.github.io/22S-ASIAAM-191A/assignments/week1/thinking_cap) if you need a reminder on how to complete the assignment.

**Comment by daikis12** [comment]
I took Geography 7 (Intro to GIS) last quarter and learned that we need to be careful when putting place names on a map. Since place names, or toponyms, can be contentious due to numerous factors such as history and politics, they should be used not to offend any particular groups or countries. This is only one of many examples of the problems with the way we use GIS, but it is important to note that mislabeling controversial geographic subjects can cause further issues and thus should be avoided. This also reminds us of the power of GIS, which can not only provide useful geographic information to users but also, if misused, increase tensions between opposing groups.

**Reply by codingdianat** [reply]
Hello,
I find the topic of map titles to be very interesting since I never gave it much thought before. I think with maps that relate to topics that are very controversial (such as covid vaccines) it is important for map makers to title the maps in a way that does not shame one group. Otherwise, their map would be viewed as biased although the map itself may not be.

**Reply by ashley-lu** [reply]
I really like how you were able to connect what we've learned so far to Geography 7 (I hope to take this class in the quarter)! Like you said, it is definitely important to be aware of how we navigate place names because they are not simply ways of indicating geography, but also ways of expressing different cultures. I wonder if there are any good ways to make our current maps less contentious on a large scale.

**Reply by joonyloony** [reply]
I really agree with this! Especially places such as the Israel/Palestine conflict region, where we are actively seeing borders and lines being redrawn to fit the settler colonial narrative, I think that when certain maps are drawn/labeled and accepted by people, it is denying the indigeneity of other peoples.

**Comment by NishuKrish** [comment]
Name: Nishaanth Krishnan
There are numerous problems with the way in which we use data and GIS in the modern world. For example, many countries make use of facial recognition data and geolocation data to monitor and track data of their citizens. This is especially problematic as governments around the world use such data to crack down on dissent, and thereby take power away from people. Additionally, our data itself can be biased due to systemic issues; there is oftentimes less data collected on communities of color. This in turn negatively influences the conclusions we make and the actions that stakeholders take. While maps/GIS and data/data science can be used to uplift and empower communities in need, there are numerous ethical issues with the way in governments and companies use these tools.

**Reply by daikis12** [reply]
Nishaanth,

You're right that data and GIS are very useful and have the ability to empower communities in need, which indicates that these tools have significant power and can foster social changes. However, as you said, the users of these powerful tools must understand what they are capable of doing with them and must not abuse their power or use them for wrong/unethical reasons. Great post!

**Reply by angelineeow** [reply]
Hi Nishaanth,

I strongly agree with you on how many countries would often overlook marginalized groups and it becomes problematic, especially in the government sector. Just like you mentioned about facial recognition, I have read articles about people from communities of color who are caught wrongly using AI due to a lack of data collection in the communities. The government definitely should consider numerous ethical issues before they use the tools. I love your response! Keep it up! :)

**Reply by lindseymardona** [reply]
Really interesting point you bring up with governments using the data of their citizens! There are definitely more than a few ethical concerns in that regard. The systemic issues playing into data collection is definitely true as well; it's something I've thought a lot about in the past few years when looking at COVID numbers.

**Comment by emmashanson** [comment]
There are a vast number of ways in which data and GIS can be misused, often to further disadvantage marginalized groups and benefit those in positions of power.  According to "An Introduction to Critical Cartography", misusing data to a way to convince people of misrepresented information has been a historical challenge, even before maps were digitized and data was a lot more "visible" in the way that it is known today, such as the times of WW2. 
For instance, cartographers even in premodern times would draw their nation as the center, often bigger or presented in a more "grandiose" manner to elevate their country's status. Now, when looking at maps drawn by people of different nationalities hundreds and thousands of years ago, there were huge discrepancies because the "outside world" was unknown, and it was easy to turn this map into an asset for a nation.  While the ethics of GIS and data are a subject of concern now, this was something that has been occurring since the beginning of cartography and has only grown since.

**Reply by daikis12** [reply]
Emma,

It is interesting to me how countries have had political intentions when creating maps of their territories. As the article mentions, cartography has always been used as a crucial political tool that illustrates countries' greatness or status and can often mislead people. This indicates the significant power that mapping/GIS possesses and that we, as people who use maps, should understand that mapmakers may have misrepresented information on their maps, especially when the maps are created by an authoritarian government.

**Reply by ashley-lu** [reply]
I like how you gave the example of cartographers drawing their own nations more centrally/larger. This only further serves to prove that cartography cannot entirely be a pure "scientific" discipline, and it can spread misinformation through misrepresentation. I also like your point about how ethics relating to GIS and data already existed since the days when maps were first made; it's something I've never thought about, but definitely true.

**Reply by kericsp** [reply]
Hi Emma, I like that you bring up the historical example of countries enlarging maps of their own territory in order to serve the political agenda of those in power in those countries. This phenomenon has current implications of inaccuracies in present-day mapping that is perpetuated in order to further political agendas, but ends up creating further confusion among the wider community or worse, among the international community.

**Comment by shubhgupta2018** [comment]
There are several problems with the application of data to GIS as it exists within the contemporary sphere. First, maps are intrinsically political and inscribe power to the nation-state rather than the communities themselves. As a result, traditional maps that focus on borders therefore perpetuate systems of settler colonialism. In addition, the drawing of borders implies that land can be possessed and owned (which in and of itself is a neoliberal conception). Furthermore, drawing borders ignores how many of these boundaries are constantly being contested in areas like Kashmir, Jammu, and Ladakh. Critical topography transcends these problems by not only inscribing maps intrinsically with power but also centering community participation over borders drawn by the nation-state and its biopolitical institutions.

**Reply by codingdianat** [reply]
Hello,
I agree that maps often perpetuate the views of those in power while disregarding many individuals. Your example of drawing borders makes me think about how the practice of Redlining in the early 20th century led to entire communities being stripped of resources-creating lasting damage to those in those communities. The people who lived in those areas were denied loans based solely on their address. This empowers the nation-state and casts asides the communities.

**Reply by emmashanson** [reply]
Hi,
I really like how you mentioned the idea of inscribing power between nation states rather than communities. My post had a similar comment on how boundaries and nations are designed for political reasons and generally have selfish intentions and leads to contested power. Despite the fact that the data comes from the people, it is used to provide greater power to those already in positions of power. Also, I really appreciated the fact that you included examples of instances of nations that currently have boundary disputes.

**Comment by ashley-lu** [comment]
I recently was able to connect what was described in the Critical Cartography reading and what we learned in class to my Middle Eastern Studies class in terms of how data and GIS perpetuate power imbalances. The term "Middle East" came to be because of Great Britain's imperialistic goals towards countries on the "Far East" such as India, with the "Middle East" acting as the space in between. Other countries also viewed it as a place to conquer and add to their own empires, especially with the collapse of the Ottoman Empire. This influenced how the Middle East came to be defined on maps, despite it not being a continental region. With the addition of the United States growing interested in the Middle East's potential for oil, the US and GB essentially used GIS to create this construct that exists in our perceptions but is not representative of the vast number of cultures, languages, and religions found in the Middle East at all. In the modern day, these problems continue to grow as US media frequently portrays the Middle East as a place of terrorism, violence, and political instability. 

Through this example, we can see that the way in which we present data can be a major problem, especially when it can misconstrue our opinions of communities in a negative way. We can also see that maps defined even a century ago can still have major implications on our perceptions today. Though some might treat cartography as wholly a science as described in Critical Cartography, it is very much subjective and influenced by political motivations. Even something that seems as simple as defining a specific region can diminish the voices that live there and empower others who are already in positions of power. In the case of the Middle East, it is almost always associated with Arabs and Islam even when there is a lot of diversity in the region. Data and GIS are powerful tools that have allowed us to see the world at a glance even without having traveled the world, but when we use them we must be careful not to assume they are definitive. Otherwise, we continue to perpetuate the power imbalances established centuries ago.

[https://daily.jstor.org/maps-created-modern-middle-east/](url)

**Reply by shubhgupta2018** [reply]
I really resonate with your response, as during AAS175B I also learned about the origins of "Middle East" as a term and its increasing replacement in academia with phrases like "Arab and Muslim-dominant communities." It also makes me think about the difference between "community" and "state," as the latter legitimizes imperialist borders. The origins of cartography, therefore, make me think deeper about the politics of naming and wonder about how we can apply that to cartography as a science and acknowledge these problematic roots.

**Comment by willa-21** [comment]
Maps cannot be considered “neutral scientific documents”, as discussed in An Introduction to Critical Cartography. In fact, maps are political in nature and “make reality as much as they represent it” (p. 15). One issue we face today is the digital divide, which means that those with ongoing access to technology (and the knowledge to use data and GIS) hold immense power in communicating knowledge and creating reality. Additionally, there is very little education surrounding data and data privacy, meaning often data is collected and used without full transparency. One example of this is during election season, maps can often lead the public to misinterpret a representation of voters. [This article ](https://www.bloomberg.com/news/articles/2020-11-03/a-complete-guide-to-misleading-election-maps)discusses how misleading maps can be. Often rural areas are shown to represent a critical mass of voters when in reality urban areas often hold far more voters. “Choropleth maps — or maps that use color, shading or patterns to show quantitative differences — are some of the most common ways to visualize how counties as well as states are voting, even though they rarely come with the important disclaimer that land doesn’t vote” (Bloomberg, 2020).

**Reply by emmashanson** [reply]
Hi,
I liked how you highlighted the reason why maps are political in nature despite the preconceived association maps have with data, math, and analytics.  The very collection of data and the way in which it is collected is political, to the point where legislation is made to regulate privacy of the individual. I also appreciate that you provided the example of election season and how even the choice of color can psychologically alter someone's impression of a political event, such as an election and voter turnout or a percentage of the nation that is choosing to elect someone.

**Reply by angelineeow** [reply]
Hi,
I agree that the maps should not be considered neutral scientific documents! I also agree with how limit our knowledge on data privacy as I only get to learn it right before I am graduating. I find it interesting from the article you mentioned that some of the maps for US elections are misleading. I did not think more about it which also proves that misleading maps create huge impacts on the general public. I think you made a good statement here! Great job!

**Comment by codingdianat** [comment]
As stated in "An Introduction to Critical Cartography", the creation of maps can often "inscribe power and support the
dominant political structures" (p 15.) As a result, many maps may not be a reflection of the truth, but rather a reflection of what a small group of people wants us to see.  For example, after the 2016 election, a series of maps went viral on Twitter that supposedly showed how the results of the election would be if only one demographic voted, resulting in a series of maps with various presidential results. What most people do not realize is that these maps are not factually accurate and result in further polarization of certain groups. The lack of media literacy leads to what [Assistant Professor Robinsion ](https://www.fastcompany.com/90230916/the-next-great-fake-news-threat-bot-designed-maps)describes as a false reliance on maps "because we’re so used to trusting them as fact." Not only does this allow for certain dominant political structures to push a false narrative, but it also allows leads to the further polarization of different groups. The abundance of maps, both accurate and inaccurate, can easily blur the line between what is factual.

**Reply by shubhgupta2018** [reply]
Your response about maps going viral on Twitter makes me think about how many of the problems with cartographical interpretation for a public audience roots itself within educational inequities. As individuals producing information for public consumption, misinterpretation is a legitimate concern, and it becomes increasingly important to make maps as easily understandable as possible. It is also important for cartographers to be transparent about the limitations of how generalizable their maps are.

**Reply by stefuntran** [reply]
It is interesting to see how social media has made a big impact on the communication of data. There are a lot of people who only spend their time looking at the post itself, and not where the data is coming from. Media literacy, or the lack thereof, is a concept that is definitely not talked about more which should be because of how information is presented through social media especially for the different generations that use it.

**Comment by antonyblin** [comment]
Maps as well as data in general are extremely impacful. According to our reading, "critical cartography assumes that maps make reality as much as they represent it", and a poorly or unethically constructed visualization of data could have disastrous effects on everything from everyday navigation to urban planning. This issue is exacerbated by the fact that "maps are no longer imparted to us by a trained cadre of experts, but along with most other information we create them as needed ourselves"; therefore, while it may be our responsibility to accurately and ethically share and visualize data, the inability to carefully regulate data visualization, especially over social media, is liable to enable potentially malicious individuals to distort people's perceptions of the world around them. Furthermore, because there exists a common misconception that visualizations of data are scientific and unbiased, viewers may spend far too little time considering their validity and reliability and entirely too much time absorbing the information presented as fact, which also thereby depersonalizes and dehumanizes the individuals that constitute the data: people are not numbers. 

![image](https://user-images.githubusercontent.com/46633905/161416445-209ff021-0d6a-476e-9298-9dd7c888e2b7.png)

**Reply by stefuntran** [reply]
Hello Antony, 
I completely agree with your point where you wrote about how a lot of people see data as scientific and unbiased, they tend to jump and react immediately afterward. This causes people to ride with the idea and use the data as evidence, which is not unbiased, to support their claims. With the help of having community-based maps and data, I hope that more people are able to take everything with a 
"grain of salt" and look at where the information is coming from.

**Reply by willa-21** [reply]
I appreciate that you brought up social media and you emphasize that "there exists a common misconception that visualizations of data are scientific and unbiased". This is so true! This reminds me of how data has been portrayed on social media over the last two years, specifically with regard to racial and social justice. While social media is particularly powerful in reaching large audiences, it enables people to share information that is easily digestible, which can often oversimplify these larger, more nuanced issues. I've found myself reflecting on when and how to share information, as often social media feels watered down and irrelevant to what's really going on in the world.

**Reply by lindseymardona** [reply]
I really like the social media angle in bringing together the importance of critical cartography in a very modern day perspective as well as the dehumanization involved. I think that it becomes very easily for the average consumer to accept information as fact when it's been so nicely condensed into a very aesthetically pleasing, square infographic, instead of acknowledging that the numbers may not necessarily be perfectly representative of the truth.

**Comment by stefuntran** [comment]
The reading "An Introduction to Critical Cartography" gave me more knowledge on a subject that I had very little knowledge about. Critical cartography is political by nature and can be shown by presenting historical events and maps that big entities such as the government have done to implement it. The notorious example is redlining, a concept that Professor Albert talked about in class. The U.S. government brought people out to survey different zip codes from all kinds of aspects such as income level and the surrounding environment such as if there were any factories near these communities. The information that the surveyors put down continues to affect these communities today because schools are underfunded in the areas that they deemed were not suitable for the best quality of life and pinned minority groups to live in these areas. This transitions to how maps can hurt communities because many Indigenous communities in America have been stripped of their native lands because the U.S. is the one who is making the maps.

**Reply by ntran27** [reply]
Hello! Your reflection made me think of the U.S. census and how often people are told to complete the survey so they can be counted/represented. Being counted does not necessarily benefit communities/individuals. You brought up a great point in saying that the information surveyors record can also be used to harm already marginalized communities. Particularly with how resources such as school funding is allocated. Researchers and surveyors often want as many people as possible to complete their surveys but there's much less done by these people to reassure the survey respondents that their information won't be misused or misinterpreted.

**Reply by ananyakodali17** [reply]
Hi! The specific point that you brought up about how surveyors putting down information in the past leading to current issues in marginalized communities explains the problems today with GIS really well, and really emphasizes the point that just because data is taken, it doesn't necessarily mean that anything will be done with it or that it will be interpreted properly to give the help needed to the areas which need it. Since the U.S. is the one making the maps, it is so easy to just ignore the impact maps have had on Indigenous communities, and GIS has definitely played a role in this.

**Comment by ntran27** [comment]
Maps, like the people who create/use them, are susceptible to outside influences such as power and politics. We are vulnerable to documenting and visualizing the world and communities around us in ways we were socialized to categorize and perceive others. In other words, how one makes use of data and GIS can depend on our lived experiences, social identities, education, and other factors. Our representation of others with labels, numbers, and color-coding likely does not represent the full breadth of life people experience daily. A critical research approach might offer counter methods for research with GIS and spatial data; however, there is subjectivity and variance in everyone's definition of "critical." This makes it more important for researchers to consider how, exactly, they are conducting and presenting their work. Concerning academia and research, there can also be pressure on the most critical scholars (particularly early-career scholars) to use data or GIS in ways that fit with dominantly scholarly discourse to make scholarship more acceptable to reviewers at the expense of the communities involved in the study. A question that I often reflect on is how to empower critical researchers to advance an agenda of resistance and center marginalized communities within this work.

**Reply by willa-21** [reply]
Such good points! Within academia (including cartography) as a whole, it seems it is rare to be able to document the full breadth of one's lived experiences, especially given the reasons you mentioned. I'm looking forward to being able to combine technology (GIS) and criticality in research (or a critical lens) to move towards centering communities and empowering individuals. Personally, I think it takes a high level of care and attention to the ways we (the researcher) hold power and privilege, and how we navigate our work from this positionality.

**Reply by ananyakodali17** [reply]
Maps are definitely easily biased depending on the creator's lived experiences and different factors, which is something that is not often considered when studying them. I agree with your statement that representing others with different quantitative labels does not fully capture their experiences and lives, but that there is always subjectivity in any method of conveying data. It is very difficult as a young academic to find data which is against what had previously been studied, as they are not yet well-established, and I think that is a great point to bring up as well!

**Comment by angelineeow** [comment]
There are many problems with the way we use data and GIS today. When we map our data, the data we obtained sometimes could be biased. Biased data is a common issue in this modern world because of the structural characteristics of the systems that produce the data. Therefore, we should not assume like in our reading where the author mentioned  "maps make reality as much as they represent it" when mapping because the visualization created from biased data would be misleading. Since GIS is used to help the readers to better understand the pattern and relationships, it is important to note that misleading visualization would only lead to undesirable effects such as the readers being fed by prejudiced results which would be a factor in complex social repercussions.

**Reply by joonyloony** [reply]
I agree! I think that GIS definitely could help with giving more context to conflict ridden zones, though I feel like right now we need technology to show it (like hovering a mouse over a region to see context) so it doesn't really help physical maps (like in elementary schools and so on). Maybe in the future we will be able to expand GIS past a screen!

**Reply by NishuKrish** [reply]
Biased data is a huge problem in the modern world. With so much data being out there, it has become increasingly difficult to determine which is accurate data and which isn't. The impacts of this are very harmful as data is often spun in a way to suit the desires of those in power. Many news articles often make use of biased data to reinforce stereotypes on developing countries, while the reality might reflect something different.

**Reply by XiongHer** [reply]
Hello, I agree with you. With everyone having smart phones, it is so easy for us to look up and consume information online, especially through social medias, that it is hard to determine what kind of information is accurate or not, or which information outlets you decided to trust to consume those information. There are so many "alternative facts" out there that GIS can easily misled people's opinion on certain social and political issues.

**Comment by joonyloony** [comment]
I learned a bit about cartography first in an Eps Sci class on Earthquakes, where people used data to map out the landscape of the ocean floors, and so I think that cartography and GIS is pretty cool in general, but I think that there are a lot of social implications from them as well when we are mapping areas that are less remote than the bottom of the ocean. From what I’ve learned from previous Asian Am classes, I think that the way a lot of maps and country borders are portrayed could be problematic as the “mainstream” map is usually upheld by the dominant/”ruling” communities which could be used to marginalize other groups. When thinking of the Israel/Palestine conflict, we can see that their borders and the maps of those regions are actively being redrawn, and we can also relate it back to colonial times when settlers began drawing maps of their colonies and claiming indigenous land as their own. A lot of times, I think that mainstream GIS mapping may potentially support a settler colonial narrative. As we can see in the reading, “maps inscribe power and support the dominant political structures,” and so we should be very conscious of that when looking at conflict ridden zones as many people accept maps as the truth.

**Reply by kris10fu** [reply]
Hi! I definitely agree that, although maps are often thought of as objective, they can be used for problematic purposes, especially in terms of politics. I also thought it was interesting how you brought up how the history of colonialism has impacted map-making because it shows how narratives of the "truth" are often impacted by biased and subjective powers. I wonder how we can make the general public more conscious of the subjectivity of maps, and if we can use GIS tools and data to help with this purpose.

**Reply by antonyblin** [reply]
Your example with the Israel/Palestine conflict is great! I had always thought of mapmaking and representations of data as more static practices that took a lot more time to develop, but mapmaking is actually a very active practice, exemplified by the constant redrawing of borders to suit particular interests. I also really like your links to colonialism; the continuity of this problem is really quite clear as a result. How would you suggest we begin to educate the general public about skepticism, especially in relation to reading maps and data?

**Comment by albertkun** [comment]
#### ---- cut off for early submissions ----

**Comment by ananyakodali17** [comment]
While current practices with using data and GIS to convey meanings and arguments are allowing people to better support their arguments, it makes it much easier to skew things in a certain way to persuade a large audience. Taking data can be very subjective and biased, which means that the created map can indicate a clear bias that is not representative of the full situation. When people see a map of data, they are more likely to believe it, as mentioned in the Critical Cartography reading which says that maps make reality as much as they represent it. Since GIS is used in political contexts, the results of the maps can have a hugely negative impact on communities that need more funding, hidden systemic issues that are never addressed, and so much more, and therefore these issues must be addressed.

**Reply by kris10fu** [reply]
Hi! I definitely agree with what you wrote about the political implications of using GIS, and how it can have a negative impact on under-served communities. I also think that, more often than not, the organizations or people that do have access to the tools for GIS are more privileged or wealthy, so that adds to the power imbalance of being able to afford the resources for creating maps. Additionally, large databases are often created by corporations or governments that have hidden agendas and biases, which makes it even more difficult to create objective maps.

**Reply by XiongHer** [reply]
Hello, I'm totally agreed with you. Interpreting data and using GIS can be very subjective and biased when we thought about who collect the data and for what purpose. In our current contested political dynamic, political maps can be very damaging to communities of color, especially during elections, that continue to perpetuate systemic racism (e.g., redlining, gerrymandering, etc.).

**Comment by kris10fu** [comment]
When using data and GIS, we often create an incomplete or biased picture of the entire issue. From previous digital humanities classes, I've learned about how missing data can lead to skewed portrayals and understandings of an issue or topic, which can lead to wrong interpretations. Acknowledging what data is missing can help provide a more accurate understanding of the issue. Additionally, data can be visualized in biased ways that can promote a certain agenda or ideology, leading others to believe that what they see is the truth. For example, the _Hands-On Data Visualization_ textbook wrote about how using inappropriately scaled graphs can lead to a misleading understanding of the scale of an issue. Furthermore, with GIS, depending on what factors and tools are used for visualization, only a partial picture of the entire topic or issue is likely to be mapped. Again, these tools can be used for political gains that prioritize the majority group over minorities, erasing valuable and important voices in the process. As discussed in the "An Introduction to Critical Cartography" reading, maps are not passive but rather active agents that can be used for knowledge, power, and social change. Thus, the process of making maps is imbued with bias and hidden agendas that depend on the purposes behind their creation.

**Reply by NishuKrish** [reply]
You bring up a great point in that the data that underlies maps and other visualizations can be skewed in order to show a particular narrative or can be misleading. One common thing I see is that many visualizations are not normalized or adjusted for population. This can be misleading as it can make certain values look bigger when in reality it is not considering the size of that region's population. For example, COVID-19 case rates may show that certain countries or states may have high case rates in absolute terms, but relatively low ones in relative terms (per capita or per 100 thousand).

**Reply by antonyblin** [reply]
I absolutely agree that the scaling and formatting of graphs and maps can negatively affect people's perception of data. I specifically remember being taught in elementary and middle school to watch out for graphs that have an axis that does not start at 0, and I think it would be especially beneficial to generalize this principle to data visualizations for younger students and teach them as they grow.

**Comment by XiongHer** [comment]
I think a problem with the way we used data and GIS stem from who is interpreting the data set and what purpose/goal they used it for. Everyone has different opinions and stories to tell, so when they interpret certain data set, there is a specific message they want to convey, especially social and political issues nowadays. Also, when people use data to represent community of color, their messages usually portray negative stories, showing gaps as to why there is a need for interventions rather than trying to understand the cause and story of the community as why that’s the case (e.g., gaps in education attainment among underrepresented students, dropout rates, retention rate, etc.). Also, GIS provides great visual representation and easy-to-assess information on social issues that can be easily misled, spread, and damage to the community. After all, I think it is important to be critical of who and how they use/interpret the data and GIS, especially today where we have easy access to the internet and information.

**Reply by kaittli** [reply]
I agree that if there are discrepancies in what is reality versus what is shown in the data and the GIS, there should be accountability as it can result in harmful consequences for the general public. Your point on communities of color being misrepresented is a multifaceted topic and could be investigated by looking at the source of the data or analyzing the characters/beliefs of the creators. These misrepresentations can have tangible effects on these communities, either by exacerbating stereotypes or feeding false information, while benefiting other communities at their expense.

**Comment by kaittli** [comment]
From the in-class reading, there is a frequent message that is being conveyed—GIS is powerful in conveying information to readers and when misused, can illustrate a false reality. Because maps are conveyors of knowledge, it is important that the details that they represent are accurate, but maps may not necessarily be a neutral messenger of information. The creators of these maps have the ability to impart their own biases as they are the people in control, and thus, can influence the accessibility of information. These people in power can decide which aspects to highlight and which aspects to completely ignore based on their own subjective opinions. Or perhaps, the people who use these maps may not be aware that these maps are inaccurate because they may have unknowingly used data that was unrepresentative/biased. Even the collectors of data can impart their own biases by focusing on aspects that they care about most, leading people to have skewed perceptions of real world phenomena upon seeing visualizations with those data. It is important to address these issues because they all have a part in the narrative that we are trying to convey.

**Reply by kericsp** [reply]
Hi! I agree that individual biases can have a significant impact on mapping and the way that the world is portrayed, but I also think you hint at a great point that the creators of maps can end up in an echo chamber of sorts, that results in the complete exclusion of people that are actually impacted by the maps. This is especially consequential given that maps don't just represent reality, they create reality. This means that a seemingly innocuous bias of an individual or a handful of researchers can end up having a dramatic impact on the mapped regions, for better or for worse.

**Reply by AHatalski** [reply]
I really like how you emphasized GIS as a tool to convey knowledge, and therefore it relies on the neutrality or biases of whoever is using the tool to create maps and such. I also see different layers of potential problems weather the data itself is biased, or the way it's visually represented could present a whole different bias.

**Comment by kericsp** [comment]
I think one of the biggest problems facing the current state of GIS is the control that large tech companies and powerful governments have over digital maps. When you think about the power that, for example, Google Maps and Yelp have over the way that its users think about the world around them and the landmarks or places that they designate as significant, it's actually pretty shocking. These corporations have the power to shape how the majority of the population thinks about the world around them, and frame areas as either desirable or undesirable. This has ripple effects that impact housing, funding, and availability of essential services in areas that are ignored or minimized by these maps. It can also intensify gentrification and perpetuate existing power structures, such as in areas that are or have been colonized. One example is how China's political clout leads to mapping that minimizes the autonomy of Taiwan, Hong Kong, and Uighur-majority areas. However, crowdsourced data and maps created by and for the people that live in affected areas is a strong start to tackling these issues.

**Reply by penelopelamb** [reply]
I like that you mentioned crowdsourced data and maps as a way to neutralize the power Google Maps has. It made me think of the navigation app Waze that uses crowdsourced data for live traffic updates, which became pretty popular a few years ago. It would take the user on funky detours to avoid any traffic, usually much different than the routes given by Google Maps. However, the app was bought out by Google. I think that the app still supposedly runs off of crowdsourced data, but I wonder to what extent Google's ownership changed the way Waze uses data.

**Comment by lindseymardona** [comment]
After reading the paper, _An Introduction to Critical Cartography_, it becomes evident as a reader that much of the narrative told by maps and GIS is driven by power. Power is not inherently bad; as stated in the reading, "power is not a negative force that must be dispelled" (p. 14). However, critique becomes more and more necessary in order to analyze the conditions under which our knowledge comes from, as well as understand where power came into play historically. In addition to the historical legacy of power, it remains present in present day usage of mapping and GIS. Access to the Internet, a PC, and the ability to learn and use software all create ongoing disparities between those with power and those without it.

**Reply by spolcharoen** [reply]
I totally agree that data and GIS are influenced by power. Especially with big corporations running the most well-known map services, it is impossible to have zero bias. On your point of critique, what are some ways you think we can fact-check these services that are commonly used?

**Comment by penelopelamb** [comment]
We use data and statistics to realize patterns in the world around us. But when the data we collect is a poor representation of reality, the results are misleading. Real data is not necessarily accurate data, and the way data is collected--where it's from, for how long and how much--has a significant impact on its accuracy. Because data is susceptible to manipulation, it can be used to present false narratives or for unethical purposes. For example, As the consumer, it's a necessary skill to be able to recognize the margin of inaccuracy any given data may have, and to know how and when to do more research.

**Reply by spolcharoen** [reply]
I agree with your point that data is subject to manipulation and that consumers must be aware of this. I believe it is also the job of big corporations running these data or GIS services to hold themselves accountable as well. Just because a consumer is aware of potential bias, that does not mean that they are knowledgeable enough to identify inaccuracies. Do you think this responsibility falls on the services or consumers or perhaps even the government?

**Reply by AHatalski** [reply]
I'm wondering about the distinction between real data and accurate data however I definitely agree that the varying quality of data has lots of potential problems. Additionally,  think its important to acknowledge that the income level of communities impacts the quality and quantity of data collected.

**Comment by spolcharoen** [comment]
As a Statistics major, I often encounter issues of bias within data. Data and GIS are powerful tools to uncover research questions and information. However, it is difficult to find and identify sources that are unbiased and truly representative of a general population. After doing the reading, it is evident that data and GIS are influenced by power, and consequently subject to bias. For example, real-world applications of GIS, such as Apple Maps and Google Maps, are run by giant corporations and are thus subject to bias based on what these corporations deem is "correct." This view might not reflect reality, which can be harmful to the communities they are representing.

**Reply by kaittli** [reply]
I like your point that people are influenced by what the corporations say about their own applications—they may say that they are correct, but then do not offer much clarity on how they developed the application or where exactly they sourced their data. Without any investigation, these corporations can get away with deceiving the general public. It is important that consumers of these GIS models are aware of potential drawbacks of using specific GIS applications or else they will vulnerable to being misled.

**Comment by AHatalski** [comment]
The most prominent problems that come to mind when thinking about the use of gis and geospatial data is the potential for misrepresentation. This could result in misleading data visualizations and deceptive maps. Additionally I keep thinking about the redlining example mentioned in class because of its long term effects. Redlining maps relied on population data from the 1930s, yet the segregation and policies those maps sparked still very much affects those communities today.
